.gitignore

```diff
@@ -4,3 +4,4 @@ rootfs/opt/router/sbin/router
 rootfs/opt/router/sbin/router.*
 vendor/
 coverage.txt
+.idea*
```

Comment:
Hiding files under .idea isn't strictly related to the ingress changes, so ideally this would be in a separate PR.